aui.views/src/AUI/View/AView.h

```diff
@@ -74,6 +74,8 @@ class API_AUI_VIEWS AView: public AObject
     friend class AViewContainer;
 private:
 
+    std::function<void(const _<AView>&)> mOnChildFocused;
```

Comment:
In AUI, signal-slot system is used for such cases.
Take a look on example of `clicked`: how it's [declared](https://github.com/aui-framework/aui/blob/f98794d864a795f534689b03545eaabc6a9baac6/aui.views/src/AUI/View/AView.h#L842) and [used](https://github.com/aui-framework/aui/blob/f98794d864a795f534689b03545eaabc6a9baac6/aui.views/src/AUI/View/AView.cpp#L365)